### ISSUE TITLE
gail performance for other MuJoCo tasks

### ISSUE THREAD

**Issue by LinBornRain** [andrewliao11/gail-tf#7]
Hi @andrewliao11 :
When applying default sets for other MuJoCo tasks(besides Hopper), It is hard to recover the gail performance as good as the results u presented. When I training HalfCheetah using HalfCheetah expert trajs (whose average returns evaluated as 5212.5), the gail results appears to be like below:
![2018-01-29 22 15 46](https://user-images.githubusercontent.com/20317695/35514686-fa2cb2fa-0541-11e8-91bb-2be290eeceb6.png)

I also have tried to adjust the default set of 'g_step' and 'd_step', it still does not work .
Can you give me some advice for  training other MuJoCo tasks.
Thank you!

**Comment by andrewliao11** [comment]
Hi @LinBornRain , 
Did you try the command for halfcheetah in the [readme](https://github.com/andrewliao11/gail-tf#results)? I re-run the process and it can still work.
Also, recently openai just found a bug in their code (they already fixed it) that the previous trpo training doesn't train the variance variable in gaussian policy. This repo might suffer this bug and I will fix it these days and do the benchmarking again.

**Comment by Feryal** [comment]
Hi,

I also tried training on RoboschoolHopper-v1 with my own expert dataset(average return around 2800) however I also get really bad performance. While it's understandable that the reward function/observation range/physics is slightly different, I tried adjusting the hyperparams and even doing a pre BC training with no luck. 

I was wondering if you've ever tried this environment or have any suggestions?

Thanks

**Comment by agarwl** [comment]
Hi @andrewliao11 ,

I am also facing a similar problem. I am not able to get good results on RoboschoolHopper-v1 even with Behaviour cloning pretraining. The orange curve is the Mujoco Hopper-v2 while the pink and red ones are for RoboschoolHopper-v1 with and without pretraining.
![hopper_roboschool_hopper](https://user-images.githubusercontent.com/12142945/35777096-ba04a6ee-099f-11e8-8948-a3279d4c0ca0.png)

**Comment by andrewliao11** [comment]
Hi @Feryal, can you provide your entropy of the policy or what hyper-params have you tried to tune the network?

**Comment by andrewliao11** [comment]
Hi @theSparta, 
in general, tuning the g_step and d_step helps a lot. Also, reinforcement learning algorithms are relatively sensitive to random seeds, so you might want to try more

**Comment by andrewliao11** [comment]
I will first close this issue. Feel free to reopen it if needed
